tests/test_surfaces.py

```diff
@@ -136,6 +137,140 @@ def test_curvature(self):
         np.testing.assert_allclose(data["curvature_k2_rho"], -1)
 
     @pytest.mark.unit
+    def test_constant_offset_surface_circle(self):
+        """Test constant offset algorithm for a circular torus."""
+        s = FourierRZToroidalSurface()
+        grid = LinearGrid(M=3, N=2)
```

Comment:
Maybe increase the resolution of the surface and then let it use the default grid? That should increase test coverage (see previous comment).